src/exceptions.py

```diff
@@ -0,0 +1,6 @@
+class InvalidArguments(Exception):
+    """ Unwanted arguments.
+    Example: When you try to use 'path' and 'value' on input field where only one is allowed."""
+
+    def __init__(self, msg):
```

Comment:
Nit: the empty `__init__` is unnecessary - the class statement + docstring is enough to make a valid class declaration.

Reply:
done :)